src/main/java/frc4388/robot/subsystems/Vision.java

```diff
@@ -0,0 +1,39 @@
+package frc4388.robot.subsystems;
+
+import edu.wpi.first.apriltag.AprilTag;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+public class Vision {
```

Comment:
As much as I love my own code, this class seems redundant and outdated given that Apriltags and Location exist